packages/create-app/index.js

```diff
@@ -80,29 +168,53 @@ async function init() {
 
   // determine template
   let template = argv.t || argv.template
-  let message = 'Select a template:'
+  let message = 'Select a framework:'
   let isValidTemplate = false
 
   // --template expects a value
   if (typeof template === 'string') {
-    const availableTemplates = TEMPLATES.map(stripColors)
-    isValidTemplate = availableTemplates.includes(template)
+    isValidTemplate = TEMPLATES.includes(template)
     message = `${template} isn't a valid template. Please choose from below:`
   }
 
   if (!template || !isValidTemplate) {
     /**
-     * @type {{ t: string }}
+     * @type {{ framework: string }}
      */
-    const { t } = await prompt({
+    const { framework } = await prompt({
       type: 'select',
-      name: 't',
+      name: 'framework',
       message,
-      choices: TEMPLATES
+      choices: FRAMEWORKS.map((f) => ({
+        name: f.name,
+        value: f.name,
+        message: f.color(f.display || f.name)
+      }))
     })
-    template = stripColors(t)
+    const frameworkInfo = FRAMEWORKS.find((f) => f.name === framework)
+
+    if (frameworkInfo.variants) {
+      /**
+       * @type {{ name: string }}
+       */
+      const { name } = await prompt({
+        type: 'select',
+        name: 'name',
+        message: 'Select a variant:',
+        choices: frameworkInfo.variants.map((v) => ({
+          name: v.name,
+          value: v.name,
```

Comment:
Maybe we should display `JavaScript` or `TypeScript` (the variant `display`) instead of the template name once the user selects the option. I think it would be more clear as feedback:
![image](https://user-images.githubusercontent.com/583075/114295121-7c425200-9aa3-11eb-8399-992381243318.png)

Reply:
Well, I actually noticed and tried, but haven't found a way to make enquirer work that way. Might need to hack it a bit?

Reply:
I thought it was a simple change, your call then. This was just a small DX improvement.

Reply:
Found the way.
![image](https://user-images.githubusercontent.com/11247099/114757003-ebfb5a00-9d8d-11eb-868d-04afc0f81134.png)

Reply:
🎉